src/power.jl

```diff
@@ -0,0 +1,27 @@
+import Base: ^
+
+function ^(M::IntervalMatrix{T}, k::Integer) where {T}
```

Comment:
This is just one way of evaluating the power of an interval matrix. Here i propose to define a lazy wrapper, `IntervalMatrixPower <: AbstractMatrix` which holds the interval matrix (`A^1`) and the power.